policy-report/kube-bench-adapter/pkg/report/create.go

```diff
@@ -40,7 +40,7 @@ func New(cisResults *kubebench.OverallControls, name string, category string) (*
 
 func newResult(category string, control *kubebench.Controls, group *kubebench.Group, check *kubebench.Check) *clusterpolicyreport.PolicyReportResult {
 	return &clusterpolicyreport.PolicyReportResult{
-		Policy:      control.Text,
+		Policy:      check.ID + " " + control.Text,
```

Comment:
Can we use a "-" for the delimiter, instead of a space?

Are there any restrictions on the length of the policy name? If so, we may need to trim the string to meet that.

Reply:
Sure thing, @JimBugwadia! Will check and update accordingly!

Reply:
I fixed the delimiter part but I am not sure if `Policy` name uses RFC1123 or RFC1035 or does it even have such restrictions?
Here's a [link](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) that I referred to.  


Thanks a lot!

Reply:
Ho @mritunjaysharma394 - you can check the CRD for this. It does not look like there are any length constraints defined:

https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/crd/v1alpha2/wgpolicyk8s.io_policyreports.yaml#L72

Reply:
Oh got it, I guess then it is good to merge then right?